src/agenda/index.ts

```diff
@@ -1,68 +1,115 @@
-import {getApiServer} from "../meta";
-import * as Oauth from "../user/oauth";
-import {ApiGetError} from "../errors";
-
-interface AgendaEntry {
-    uuid: string;
-    time: number;
-    title: string;
-    description: string;
-}
-
-export const getAgenda = async (): Promise<Array<AgendaEntry>> => {
-    const response = await fetch(`${getApiServer()}/agenda/`, {
-        method: 'GET',
-        headers: {
-            'Content-Type': 'application/json',
-        },
-    });
-
-    if (response.status !== 200) {
-        throw new ApiGetError('Unable to get agenda');
-    }
-
-    return (await response.json()) as Array<AgendaEntry>;
-};
-
-export const getAgendaElement = async (uuid: string): Promise<AgendaEntry> => {
-    const response = await fetch(`${getApiServer()}/agenda/${uuid}`, {
-        method: 'GET',
-        headers: {
-            'Content-Type': 'application/json',
-        },
-    });
-
-    if (response.status !== 200) {
-        throw new ApiGetError('Unable to get agenda element');
-    }
-
-    return (await response.json()) as AgendaEntry;
-};
-
-export const createAgendaEntry = async (title: string, description: string, event_uuid: string, time: number) => {
-    const response = await fetch(`${getApiServer()}/agenda`, {
-        method: 'PUT',
-        headers: {
-            'Content-Type': 'application/json',
-            'X-Phoenix-Auth': await Oauth.getToken(),
-        },
-        body: JSON.stringify({
-            title,
-            description,
-            event_uuid,
-            time
-        })
-    })
-    return response.status === 200
-}
-
-export const deleteAgendaEntry = async (uuid: string) => {
-    const response = await fetch(`${getApiServer()}/agenda/${uuid}`, {
-        method: 'DELETE',
-        headers: {
-            'Content-Type': 'application/json',
-            'X-Phoenix-Auth': await Oauth.getToken()
-        }
-    })
-    return response.status === 200
+import {getApiServer} from "../meta";
+import * as Oauth from "../user/oauth";
+import {ApiGetError} from "../errors";
+
+interface AgendaEntry {
+    uuid: string;
+    time: number;
+    title: string;
+    description: string;
+}
+
+export const getAgenda = async (): Promise<Array<AgendaEntry>> => {

```

Comment:
AgendaEntry[] is "newer"

Reply:
So you want the code to be like this?
`export const getAgenda = async (): Promise<Array<AgendaEntry[]>> => {}`